src/data/blog/neontribe-joins-dxw.md

```diff
@@ -0,0 +1,55 @@
+title: 'Neontribe joins dxw'
+slug: 'neontribe-joins-the-dxw-family'
+image: null
+featured: 0
+page: 0
+status: 'published'
+language: 'en_UK'
+meta_title: null
+meta_description: null
+author_id: 1
+created_at: '2023-01-11T09:00:00.000Z'
+created_by: 2
+updated_at: '2023-01-11T09:00:00.000Z'
+updated_by: 1
+published_at: '2023-01-11T09:00:00.000Z'
+published_by: 21
+visibility: 'public'
+mobiledoc: null
+amp: null
+author_name: 'Neontribe'
+author_slug: 'neontribe'
+author_image: 'images/2019/03/neonround.png'
+author_cover: null
+author_facebook: null
+author_twitter: null
+---
+
+## Neontribe joins dxw in a move that will strengthen both agencies’ digital offers to third sector and public good organisations
+
+11 January 2023 – [dxw](https://www.dxw.com/), an employee-owned digital agency that designs, builds and runs digital public services today announces that [Neontribe](https://www.neontribe.co.uk/) has joined the dxw family.
+
+Neontribe is a digital agency with 15 years of expertise in the third sector and, like dxw, is committed to delivering digital services that make a difference. Having developed a successful partnership and worked together on projects including the recent creation of a new user-centred website for The Judiciary, the time was right for Neontribe to join dxw.
+
+As agencies with aligned values, similar ways of working and a shared commitment to their people, this transition unlocks new opportunities to reach organisations working for public good. In coming together, dxw and Neontribe will share expertise and resources to increase their impact and deliver more digital services that improve lives.
+
+Dave Mann, CEO at dxw comments: “Being employee-owned gives us the freedom and opportunity to join forces with talented like-minded organisations. With Neontribe on board, we believe we are even better placed to pursue our mission to improve public services and help organisations enhance the way they work. dxw and Neontribe share the same values based on a collective culture and vision to improve the lives of people who rely on public and third sector services, supported by a user-centred and agile approach. We are stronger together and excited about what the future holds.”
+
+With a well established and trusted reputation in the third sector, Neontribe will retain its brand identity and working practices with clients going forward. The founders and directors of Neontribe will continue working as part of the agency team, which will operate alongside dxw’s existing Delivery, GovPress and Strategy units.
+
+Harry Harrold, Director at Neontribe, added: “Having worked with dxw over the years, we know the quality of the work we can do together. We understand how to create and run multidisciplinary teams that make an impact. Working in the best interests of service users is at the core of everything we do and, as part of the dxw family, we are even better placed to make that happen and help those who need it most.”
+
+[ENDS]
+
```

Comment:
Did we want this bit? Or maybe just not the `[ENDS]` and `notes to editors` bits @harryharrold ?

Reply:
Unless `[ends]` is some magic markup, I might also elide it?

Reply:
It's a press release, if I'd thought I'd have written something more personal, but it's fine as it is.